src/components/themed/FilledTextInput.tsx

```diff
@@ -197,7 +198,7 @@ export const FilledTextInput = React.forwardRef<FilledTextInputRef, FilledTextIn
   function clear(): void {
     if (inputRef.current != null) inputRef.current.clear()
     if (onChangeText != null) onChangeText('')
```

Comment:
You never set `valueRef` to `''` inside the `clear` method. You could either do it explicitly, like `valueRef.current = ''`, or you could change the call to `onChangeText` to `handleChangeText` as you do in the other component.